selector.go

```diff
@@ -0,0 +1,35 @@
+package myradio
+
+const (
+	// Values for the current selection/where it was selected from
+	SelectorStudio1 = 1
+	SelectorStudio2 = 2
```

Comment:
Copied from PHP implementation, but would be open to suggestions on how to make this less awful

Reply:
Also Codacy hates these comments, but I think an individual comment for each one of these isn't helpful to anyone

Reply:
if they were sequential, you could've used `iota`, ...but they're not. I see no disadvantage to explicitly numbering them